src/utils/simplify-twitch-response.ts

```diff
@@ -0,0 +1,5 @@
+import { TwitchResponse } from '../interfaces/responses/twitch-response';
+import { TwitchVideo } from '../classes/twitch-video';
+
+export const simplifyTwitchResponse = (video: TwitchResponse): TwitchVideo =>
```

Comment:
This is a bit of a nitpick, but we can likely rename this function and the youtube one since they no longer really "simplify" anything but they map the response to a class instance. Do you think we should update the naming?

Reply:
Yeah, the naming can confuse, I will try to change the function name.